sim/rogue/ghostly_strike.go

```diff
@@ -0,0 +1,59 @@
+package rogue
+
+import (
+	"time"
+
+	"github.com/wowsims/wotlk/sim/core"
+	"github.com/wowsims/wotlk/sim/core/proto"
+	"github.com/wowsims/wotlk/sim/core/stats"
+)
+
+func (rogue *Rogue) registerGhostlyStrikeSpell() {
+	if !rogue.Talents.GhostlyStrike {
+		return
+	}
+
+	actionID := core.ActionID{SpellID: 14278}
+	// FIXME: Add Ghostly Strike Glyph
+	baseCost := 40.0
+	refundAmount := baseCost * 0.8
```

Comment:
Unless this was changed in WotLK, Ghostly Strike has no refund.

Reply:
Double checked and it does have the refund power on miss tag